database/migrations/2017_03_13_212917_CreateSharedExpenseTable.php

```diff
@@ -18,6 +18,7 @@ public function up()
             $table->float('amount_owed');
             $table->string('secondary_username');
             $table->string('comments');
+            $table->timestamp('date_added');
```

Comment:
Yes, I would like that. Although, you could have a settled boolean in expenses.
select * from expenses where settled = false is my first idea, but this works too. We can do joins for the graphs.